python/lsst/ip/isr/linearize.py

```diff
@@ -753,6 +756,14 @@ def __call__(self, image, **kwargs):
         """
         splineCoeff = kwargs['coeffs']
         centers, values = np.split(splineCoeff, 2)
+        if centers[0] != 0.0 or values[0] != 0:
```

Comment:
Is this deliberately `or` rather than a nested `if` as in `cp_pipe`?

Reply:
I thought it was correct as there are two issues to correct, and then discovered that giving the interpolation code centers like `[0.0, 0.0, 1.0, 2.0, 3.0, 4.0, 5.0]` causes an error even if the values for the two `0.0` points is identical.  I've split the two issues into separate checks, which should prevent that interpolation error.